Mongoose/Project 1/app.js

```diff
@@ -0,0 +1,90 @@
+const { json } = require('express');
+const mongoose = require('mongoose');
+
+
+const connectDb = ()=>{
+        return mongoose.connect("mongodb://localhost:27017/Temp")
+}
+
+const readData = async ()=>{
+    let DB = await connectDb();
+
+    let studentSchema = new mongoose.Schema({
+        name : { type:String, required:true, trim:true},
+        salary : { type:mongoose.Decimal128, required:true},
+        age : { type:Number,required:true, min:18, max:50},
+    });    
+
+    const studentModel = mongoose.model('Student',studentSchema)
+    let data = await studentModel.find()
+    console.log(data)
+    
+}
+
+
+const insertData = async ()=>{
+
+    let DB = await connectDb();
+
+    // Defining Model
+    let studentSchema = new mongoose.Schema({
+        name : { type:String, required:true, trim:true},
+        salary : { type:mongoose.Decimal128, required:true},
+        age : { type:Number,required:true, min:18, max:50},
+    });
+
+    //Compiling Schema
+    const studentModel = mongoose.model('Student',studentSchema)
+    const studentDoc = studentModel({
+        name:"RK",
```

Comment:
Should get the details from the request body, instead of hardcoding them

Reply:
Updated